src/schemathesis/runner/__init__.py

```diff
@@ -164,6 +164,7 @@ def execute_from_schema(
             except Exception as error:
                 status = Status.error
                 result.add_error(error)
+            result.seed = getattr(test, "_hypothesis_internal_use_generated_seed", None)
```

Comment:
It will be better to set it when `result` is created

Reply:
`test._hypothesis_internal_use_generated_seed` does not exist at that moment. Hypothesis creates is during test execution

Reply:
Got it, could you add a comment here?